djangocms_version_locking/helpers.py

```diff
@@ -87,22 +87,22 @@ def placeholder_content_is_unlocked_for_user(placeholder, user):
 
 def create_version_lock(version, user):
```

Comment:
Should this also be renamed to reflect that the helper is getting or creating? i.e. get_or_create_version_lock